main.py

```diff
@@ -33,7 +33,7 @@ def relais_off():
     GPIO.setup(RELAIS_PIN, GPIO.IN)
 
 
-if __name__ == "main":
+if __name__ == "__main__":
     print ("STRG+C to quit")
     try:
```

Comment:
gute idee. 
bitte den inhalt des try accept blocks in eine main methode schreiben und diese dann unter der if-bedingung aufrufen